setup.py

```diff
@@ -32,7 +32,7 @@
         "incremental",
         # See https://github.com/twisted/treq/issues/167
         # And https://twistedmatrix.com/trac/ticket/9032
-        "twisted[tls]!=17.1.0",
+        "twisted[tls]>=17.9.0",
```

Comment:
Do you want this change here or in #151?  Also, I'm not sure the comments above are now relevant.  The constraint is now primarily due to the name resolver behavior.

Reply:
I want the change in https://github.com/LeastAuthority/txkube/pull/151 .  I just pulled it into this branch because it was interrelated.  I wanted to get your feedback separately on bumping the Twisted version in https://github.com/LeastAuthority/txkube/pull/151 because at the time I wasn't sure how many versions of Twisted you wanted to maintain compatibility with.  We had a short conversion on IRC later where you mentioned you were OK with bumping the Twisted version.

If you can, please review https://github.com/LeastAuthority/txkube/pull/151 separately,and once we beat that into shape, I will pull that into this branch.